bids/modeling/statsmodels.py

```diff
@@ -698,6 +698,9 @@ def _build_contrasts(self, unique_in_contrast=None):
                 conditions = col_names
 
             for col_name in conditions:
+                if col_name == "intercept":
```

Comment:
I'm worried that this might end up over-correcting. I think we basically need a unit test that shows that:

```YAML
{
  ...,
  "Model": {
    "X": ["a", 1]
  },
  "Contrasts": [
    {"Name": "a", "ConditionList": ["a"], "Weights": [1], "Test": "t"},
    {"Name": 1, "ConditionList": [1], "Weights": [1], "Test": "t"}
  ]
}
```

Produces identical `ContrastInfo`s as each of

```YAML
{
  ...,
  "Model": {
    "X": ["a", 1]
  },
  "DummyContrasts": {"Test": "t"}
}
```

```YAML
{
  ...,
  "Model": {
    "X": ["a", 1]
  },
  "DummyContrasts": {"Contrasts": ["a", 1], "Test": "t"}
}
```

At both the run and subject levels.